Frends.PDF.Create/Frends.PDF.Create/Definitions/DocumentContent.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.ComponentModel;
+
+namespace Frends.PDF.Create.Definitions

```

Comment:
I don't mind that much, but I think namespaces can be one-lines now in newer .NET, so you could easily eliminate one level of indentation :)

Reply:
Changed namespaces to one-line namespace definitions.